test/land/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# test for creating pseudo ensemble for use in letkfoi
+add_test(NAME test_gdasapp_letkfoi_create_ens
```

Comment:
can you rename this test to make it clear it is land/snow DA. I suggest `test_gdasapp_land_create_ens` rather than `letkfoi`